operator_ui/package.json

```diff
@@ -81,10 +73,22 @@
     "redux-object": "^0.5.6",
     "redux-thunk": "^2.3.0",
     "serve": "^10.0.0",
+    "typescript": "^3.5.0",
     "url": "^0.11.0",
     "use-react-hooks": "^1.0.7",
-    "uuid": "^3.3.2",
-    "typescript": "^3.5.0"
+    "uuid": "^3.3.2"
   },
-  "devDependencies": {}
+  "devDependencies": {
```

Comment:
Good that this process has started. Ideally, isn't the babel family (+cross-env, and maybe few other) a devdependency too? Perhaps they should be extracted in this PR?

Reply:
All the test deps and tooling deps should be extracted into `devDeps` but I'd reckon we should do this in a separate PR since it doesn't pertain to typescript in particular